### PR TITLE
Updated HISTORY.md

### PR DESCRIPTION
**Pull request by Vedant1202**
## Description
Updated HISTORY.md with: 

- Title and description
- Index
- Previous releases
- Links to mentioned PRs
- Typos, and punctuation error fixes

## Motivation and Context
- Extends [#390 ](https://github.com/camicroscope/caMicroscope/issues/390).

## How Has This Been Tested?
- Tested on Ubuntu 18.04, Firefox

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by birm** [comment]
This must have been a lot of very boring work. I'm unsure about the index section, thoughts?

**Comment by Vedant1202** [comment]
@birm, Umm, the index section was just an addition from me since the document got too long. So I thought the index would be helping to navigate easily through various releases instead of scrolling multiple times.

**Comment by Vedant1202** [comment]
Oh, I'm sorry I did'nt notice that the index links were not working. Apparently, github markdown removes any periods '.' in the headers, as mentioned in this [comment](https://gist.github.com/asabaylus/3071099#gistcomment-1593627). I've fixed it now.

File: HISTORY.md [hunk 1]
```diff
@@ -1,67 +1,231 @@
...
+* Better UI/colors for viewing annotations
+* Disabled creation of annotations
+* Viewing GEOJSON annotations in new UAIM3 format
+* Creating Rectangles and FreeHand Polygons in UAIM3 format
+
 
-### Older Versions
-see the [release log](https://github.com/camicroscope/caMicroscope/releases)
+### Other Details
+Refer the [release log](https://github.com/camicroscope/caMicroscope/releases).
```
Comment (birm) [inline HISTORY.md:231]:
```suggestion
Refer to the [release log](https://github.com/camicroscope/caMicroscope/releases).
```

**Pull request merged**